api/environments/dynamodb/dynamodb_wrapper.py

```diff
@@ -5,24 +5,34 @@
 from boto3.dynamodb.conditions import Key
 from django.conf import settings
 from django.core.exceptions import ObjectDoesNotExist
-from flag_engine.api.document_builders import build_identity_document
+from flag_engine.api.document_builders import (
+    build_environment_document,
+    build_identity_document,
+)
+
+from environments.models import Environment
 
 if typing.TYPE_CHECKING:
     from environments.identities.models import Identity
 
 
-class DynamoIdentityWrapper:
+class DynamoWrapper:
+    TABLE_NAME: str = None
+
     def __init__(self):
         self._table = None
-        if settings.IDENTITIES_TABLE_NAME_DYNAMO:
-            self._table = boto3.resource("dynamodb").Table(
-                settings.IDENTITIES_TABLE_NAME_DYNAMO
-            )
+        table = getattr(settings, self.TABLE_NAME)
```

Comment:
Based on this logic, I'm not sure that `table_name` is the correct attribute here, it's the name of the setting that holds the table name, right? So should it be `table_name_settings_attribute_name` perhaps? 

```python
table_name = getattr(settings, self.table_name_settings_attribute_name)
```